task3_2.py

```diff
@@ -0,0 +1,21 @@
+# Напишите программу, которая найдёт произведение пар чисел списка.
+# Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+# Пример: [2, 3, 4, 5, 6] => [12, 15, 16]; [2, 3, 5, 6] => [12, 15]
+
+
+import random
+
+lst = [random.randint(0, 10) for i in range(random.randint(5, 10))]
+print(f"Исходный список:\n {lst}")
+# print(len(lst))
+
+
+def mult(list):
+    mult = []
+    for i in range((len(lst) + 1) // 2):  # Мое мнение, что при не четном списке средний элемент не надо учитывать!!!
+    # for i in range(len(lst) // 2):      # Вот это решение, опять же это мое мнение, я считаю правильным!!!
+        mult.append(lst[i] * list[len(lst) - 1 - i])

```

Comment:
```suggestion
        mult.append(lst[i] * lst[-1 - i])
```
Если сделать так, что ваш первоначальный вариант верный, а `for i in range(len(lst) // 2)` уже верным не будет.